ebuild-writing/misc-files/patches/text.xml

```diff
@@ -37,6 +37,12 @@ here. Mirrored patches are often compressed with <c>xz</c> or
 Patches included in <c>${FILESDIR}</c> should never be compressed.
 </note>
 
+<warning>
+Starting from EAPI=6 a strip patch level was limited to the <c>-p1</c>.
+Although it can be overridden with a <c>eapply -p&lt;strip_level&gt;</c> command
+it is higly recommended to enforce portage defaults.
```

Comment:
This is not a "portage default" in particular.
Maybe "... recommended to adapt the patch itself to work with the -p1 default."?

Reply:
@ulm, wfm, pushed